Jenkinsfile

```diff
@@ -2,7 +2,7 @@ node('linux && docker') {
   checkout scm
   def commitHash = sh(script: 'git rev-parse HEAD', returnStdout: true).trim()
   def tag = sh(script: "git tag --contains", returnStdout: true).trim()
-  def isBump = tag ==~ /^v[0-9].*$/
+  def isBump = !!tag
```

Comment:
I simplified here because I could not figure out why the `==~` would not return true for the regex `/[0-9].*$/` when the tag is:

```
@coveo/atomic@0.32.0
@coveo/headless@1.0.0
@coveo/quantic@0.32.0
headless-react@0.32.0
```